daemon/execdriver/native/tmpfs.go

```diff
@@ -0,0 +1,61 @@
+package native
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func genTmpfsPremountCmd(tmpDir string, fullDest string, dest string) []configs.Command {
+	var premount []configs.Command
+	tarPath, err := exec.LookPath("tar")
+	if err != nil {
+		logrus.Warn("tar command is not available for tmpfs mount: %s", err)
+		return premount
+	}
+	if _, err := exec.LookPath("rm"); err != nil {
+		logrus.Warn("rm command is not available for tmpfs mount: %s", err)
+		return premount
+	}
+	tarFile := fmt.Sprintf("%s/%s.tar", tmpDir, strings.Replace(dest, "/", "_", -1))
+	if _, err := os.Stat(fullDest); err == nil {
+		premount = append(premount, configs.Command{
+			Path: tarPath,
+			Args: []string{"-cf", tarFile, "-C", fullDest, "."},
+		})
+	}
+	return premount
+}
+
+func genTmpfsPostmountCmd(tmpDir string, fullDest string, dest string) []configs.Command {
+
+	var (
+		postmount []configs.Command
+		err       error
+		tarPath   string
+		rmPath    string
+	)
+	tarPath, err = exec.LookPath("tar")
```

Comment:
I cleaned it up, but I don't believe I can use the if x, err := 
notation since I need to use x later in the code.

When I use
```
	if tarPath, err := exec.LookPath("tar"); err != nil {
		return postmount
	}
	if rmPath, err := exec.LookPath("rm"); err != nil {
		return postmount
	}
```
I end up with

```
.gopath/src/github.com/docker/docker/daemon/execdriver/native/tmpfs.go:50: undefined: tarPath
.gopath/src/github.com/docker/docker/daemon/execdriver/native/tmpfs.go:54: undefined: rmPath
```